examples/index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <title>Example</title>
+
+    <link rel="stylesheet"
+			type="text/css"
+			href="https://necolas.github.io/normalize.css/4.1.1/normalize.css" />
+
+    <link rel="stylesheet"
+			type="text/css"
+      href="https://fonts.googleapis.com/css?family=Roboto" />
+      
+    <link rel="stylesheet"
+      type="text/css"
+      href="../css/SmartSelect.css" />
+    
+    <style type="text/css">
+			* , *:before, *:after{ 
+				box-sizing:border-box; 
+				-moz-box-sizing:border-box; 
+				-webkit-box-sizing:border-box; 
+				-ms-box-sizing:border-box;
+			}
+    </style>
+  </head>
+  <body>
+    <div class="container">
+      <div id="example">
+      </div>
+    </div>
+    <script src="./Compiled/example.js"></script>
+    <script>
+      Elm.SmartSelectExample.Main.init({node: document.getElementById("example")});
```

Comment:
I think it makes sense to split this into two apps (one for each example) to make each example easier to understand and make it easier to see what's really needed to get this running.

Reply:
Agreed. Will do.